### PR TITLE
Adding Shapefile

### PR DESCRIPTION
**Pull request by BZ-BowenZhang**
This pull request includes: 

1. Adding a Geofile folder under the current data folder, including the shapefiles of NUT1 boundaries of the UK.
2. Adding a paragraph in the `readme.md` from line 100-102 to introduce the geographical dataset

### CODE REVIEW

**Comment by BZ-BowenZhang** [comment]
This pull request is linked to issue #86 for adding the shapefiles.

File: README.md
Comment (griff-rees) [inline README.md:1]: Just a suggestion for the text:

```md
The geographical dataset can be used for visualising climate data. It mainly includes administrative boundaries published by the Office for National Statistics (ONS). The dataset is sharable under the [Open Government Licence v.3.0](https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/) and is available for download via this [link](https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/NUTS_Level_1_January_2018_FCB_in_the_United_Kingdom_2022/FeatureServer/replicafilescache/NUTS_Level_1_January_2018_FCB_in_the_United_Kingdom_2022_7279368953270783580.zip). We include a copy in the `data/Geofiles` folder for convenience.
```

**Review by griff-rees — CHANGES_REQUESTED** [review]
Looks good to me. Just made a minor suggestion for the `README.md` text, mainly a link to the Open Government Licence. Only other thought is whether the shape of London mentioned in #68 needs any further checks.

File: README.md
Comment (BZ-BowenZhang) [reply README.md:1]: That sounds good! Thanks Griff!

**Comment by BZ-BowenZhang** [comment]
> Looks good to me. Just made a minor suggestion for the `README.md` text, mainly a link to the Open Government Licence. Only other thought is whether the shape of London mentioned in #68 needs any further checks.

The London shapefile actually is a clip from the NUT1 data, so it is from exactly the same source.

**Comment by BZ-BowenZhang** [comment]
@griff-rees  I revised the `README.md`, and also added a sentence to clarify the `three.cities` folder.

**Review by gmingas — APPROVED** [review]
I think this is good to go

**Review by griff-rees — APPROVED** [review]

**Pull request merged**